src/cyc_limits.h

```diff
@@ -5,14 +5,19 @@
 #include <limits>
 
 namespace cyclus {
+
+/// generic epsilon values
+extern double cy_eps;
```

Comment:
I know... 
I have tried to put the primary definition in cy_limits.h, but as it is an header file it fails at compilation/link (multiple includes...)

There is maybe a better way to do this but I don't know how...
I was trying to define a global variable in a header file, and I fail finding a good solution. As far as I understood, my variable can be:
- **static**
- **extern**

the problem with the **static** is each call to the variable make a local copy of it... so I'll not be able to change it globally as I want,
With the **extern**, I have to redeclare somewhere my variable, which I can not do in a header file, because of linking issue...

I put the initial declaration in context.cc because it is the place where the SimInfo is defined, and this eps variables are also stored there...